src/server/actions/study-actions.ts

```diff
@@ -26,7 +26,7 @@ export const fetchStudiesForMember = async (memberIdentifier: string) => {
             'study.researcherId',
             'study.status',
             'study.title',
-            'user.name as researcherName',
+            'user.fullName as researcherName',
```

Comment:
in general should we just prefer using fullName over firstName everywhere? for my own knowledge going forward 😄

Reply:
yes, I think so, unless there's a table where we want to sort by lastName column or something like that